app/deployments/tasks.py

```diff
@@ -46,28 +54,89 @@ def update_values_for_timeseries(timeseries):
                     },
                     exc_info=True,
                 )
+                return
 
             elif error.response.status_code == 500:
+                url = error.request.url
+
+                response_500 = requests.get(url)
+
+                if "nRows = 0" in response_500.text:
+                    logger.info(
+                        f"500 error loading {timeseries[0].dataset.name} with constraint {timeseries[0].constraints} because there are no matching rows"
+                    )
+                    return
+
+                if "is outside of the variable" in response_500.text:
+                    try:
+                        times_str = response_500.text.rpartition("actual_range:")[
+                            -1
+                        ].rpartition(")")[0]
+                    except AttributeError as e:
+                        logger.warning(
+                            f"Unable to access attribute of out of data dataset {timeseries[0].dataset.name} with constraint {timeseries[0].constraints}",
+                            extra={
+                                "timeseries": timeseries,
+                                "constraints": timeseries[0].constraints,
+                            },
+                            exc_info=True,
+                        )
+                        return
+
+                    times = []
+                    for potential_time in times_str.split(" "):
+                        try:
+                            time = parse_time_string(potential_time)
+                            times.append(time[0])
+                        except DateParseError:
+                            pass
+                    times.sort(reverse=True)
+
+                    try:
+                        end_time = times[0]
+                    except IndexError:
+                        logger.warning(f"Unable to parse datetimes in error")
```

Comment:
Codacy found an issue: [Using an f-string that does not have any interpolated variables (f-string-without-interpolation)](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=6285790)